config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
+    CHECK (password IS NOT NULL OR bcrypt_password IS NOT NULL),
+    email character varying(254) NOT NULL,
```

Comment:
I increased the value to match the maximum email length.
More information here: https://stackoverflow.com/a/574698

Reply:
Please update to an alter statement. This seems like a good catch. Would you have any objection to adding a quick integration test to demonstrate this problem and fix?

Reply:
Please update to be an alter statement so this can be applied to the live schema.

Reply:
Having this file full of alter statements wouldn't work, not to mention that CREATE TABLE throws an exception if the table already exists.
This file is only being used by travis AFAIK, so this wouldn't affect the lobby at all.
This reminds me of what the NodeBB guys do: They have a folder called upgrades, which has subfolders with the version numbers.
Whenever you run `./nodebb upgrade` all the scripts that were added since the last execution of this command are run. (They update DB schemes etc.)
Adding integrationTests would be no problem.

Reply:
> This file is only being used by travis AFAIK, so this wouldn't affect the lobby at all.

Reply:
> This file is only being used by travis AFAIK, so this wouldn't affect the lobby at all.

The command travis uses is intended to be what we run to update production. If travis does not mirror prod in this way, it is pointless and bad to have travis doing its own thing. It tests nothing, adds false confidence, and is a waste of time to be maintaining.

> This reminds me of what the NodeBB guys do: They have a folder called upgrades, which has subfolders with the version numbers.
Whenever you run ./nodebb upgrade all the scripts that were added since the last execution of this command are run. (They update DB schemes etc.)

That is basically the idea. For now I was thinking our scripts would be idempotent. I had previously brought up using liquibase to do this for us but was asked to keep it simple in terms of tech stack. Hence numbered files with idempotent commands.

Reply:
`CREATE TABLE` > ` CREATE TABLE  if not exists`.

Considering that, it seems we should only run incrementals. Meaning add these statements to a '002' file.

Reply:
Good idea!